internal/controller/streams.go

```diff
@@ -211,18 +211,28 @@ func (sm *streamManager) StreamLogs(ctx context.Context) error {
 	return nil
 }
 
-// StreamCiliumNetworkFlows handles the cilium network flow stream.
-func (sm *streamManager) StreamCiliumNetworkFlows(ctx context.Context, ciliumNamespace string) error {
+// hubbleRelayDiscoery checks if hubble relay exists and if it does it returns a *CiliumFlowCollector
+func (sm *streamManager) hubbleRelayDiscovery(ctx context.Context, ciliumNamespace string) (*CiliumFlowCollector, bool) {
```

Comment:
This method name should start with a verb.
```suggestion
func (sm *streamManager) findHubbleRelay(ctx context.Context, ciliumNamespace string) (*CiliumFlowCollector, bool) {
```